### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Goulartvic**
## What is the current behavior?
Old README documentation

## What is the new behavior?
Update README documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
+|-|-|-|
+| **`faceImageCreatedEvent`** | `count: number, total: number, imagePath: String` | Emit when the camera save an image face.  
+| **`faceDetectedEvent`** | `faceDetected: Boolean` | Emit when a face is detected or hided.  
+| **`endCaptureEvent`** | - | Emit when the number of images saved is equal of the number of images set.   
+| **`barcodeScannedEvent`** | `content: String` | Emit content when detect a barcode.   
+| **`errorEvent`** |`error: String` | Emit message error.  
+| **`messageEvent`** | `message: String` | Emit message.   
+| **`permissionDeniedEvent`** | - | Emit when try to `startPreview` but there is not camera permission.
     
 ## License
```
Comment (TeruyaHaroldo) [inline README.md:63]: Remove this entire session.

File: README.md [hunk 2]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
 ## API
 
 Describe your plugin methods and properties here. See [nativescript-feedback](https://github.com/EddyVerbruggen/nativescript-feedback) for example.
```
Comment (TeruyaHaroldo) [inline README.md:31]: Unnecessary line.

File: README.md [hunk 3]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
-| some property | property default value | property description, default values, etc.. |
-| another property | property default value | property description, default values, etc.. |
+
+## Methods   
+  
+| Function | Parameters | Return Type | Valid values | Description |
+|-|-|-|-|-|  
+| **`hasCameraPermission`** | - | boolean | - | Return if application has camera permission.
+| **`startPreview`** | - | void | - | Start camera preview if has permission.
+| **`startCaptureType`** | `captureType : String` | void | `none` default capture type. `face` for face recognition. `barcode` to read barcode content. | Set capture type none, face or barcode.
```
Comment (TeruyaHaroldo) [inline README.md:39]: Is it not `string`?

File: README.md [hunk 4]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
+| **`stopCapture`** | - | void | - | Stop any type of capture.
+| **`toggleCameraLens`** | - | void | - | Set camera lens facing front or back.
+| **`getCameraLens`** | - | number | - | Return `number` that represents lens face state: 0 for front 1 for back camera.  
+| **`setFaceNumberOfImages`** | `faceNumberOfImages: number` | void | Any positive `number` value | Default value is 0. For value 0 is saved infinity images. When saved images reached the "face number os images", the `onEndCapture` is triggered.
+| **`setFaceDetectionBox`** |`faceDetectionBox: Boolean` | void | `True` or `False` | Set to show face detection box when face detected.
```
Comment (TeruyaHaroldo) [inline README.md:44]: Is it not `boolean`?

Comment (TeruyaHaroldo) [inline README.md:44]: Is it not "`true` or `false`"?

File: README.md [hunk 5]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
+| **`setFacePaddingPercent`** | `facePaddingPercent: number` | void | Any positive `number` value | Set face image and bounding box padding in percent.  
+| **`setFaceImageSize`** | `faceImageSize: number` | void | Any positive `number` value | Set face image size to be saved.    
+  
+<br/>  
+  
+## Events
+
+| Event | Parameters | Description |
+|-|-|-|
+| **`faceImageCreatedEvent`** | `count: number, total: number, imagePath: String` | Emit when the camera save an image face.
```
Comment (TeruyaHaroldo) [inline README.md:55]: Is it not `string`?

File: README.md [hunk 6]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
+| **`setFaceImageSize`** | `faceImageSize: number` | void | Any positive `number` value | Set face image size to be saved.    
+  
+<br/>  
+  
+## Events
+
+| Event | Parameters | Description |
+|-|-|-|
+| **`faceImageCreatedEvent`** | `count: number, total: number, imagePath: String` | Emit when the camera save an image face.  
+| **`faceDetectedEvent`** | `faceDetected: Boolean` | Emit when a face is detected or hided.
```
Comment (TeruyaHaroldo) [inline README.md:56]: Is it not `boolean`?

File: README.md [hunk 7]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
+<br/>  
+  
+## Events
+
+| Event | Parameters | Description |
+|-|-|-|
+| **`faceImageCreatedEvent`** | `count: number, total: number, imagePath: String` | Emit when the camera save an image face.  
+| **`faceDetectedEvent`** | `faceDetected: Boolean` | Emit when a face is detected or hided.  
+| **`endCaptureEvent`** | - | Emit when the number of images saved is equal of the number of images set.   
+| **`barcodeScannedEvent`** | `content: String` | Emit content when detect a barcode.
```
Comment (TeruyaHaroldo) [inline README.md:58]: Is it not `string`?

File: README.md [hunk 8]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
+  
+## Events
+
+| Event | Parameters | Description |
+|-|-|-|
+| **`faceImageCreatedEvent`** | `count: number, total: number, imagePath: String` | Emit when the camera save an image face.  
+| **`faceDetectedEvent`** | `faceDetected: Boolean` | Emit when a face is detected or hided.  
+| **`endCaptureEvent`** | - | Emit when the number of images saved is equal of the number of images set.   
+| **`barcodeScannedEvent`** | `content: String` | Emit content when detect a barcode.   
+| **`errorEvent`** |`error: String` | Emit message error.
```
Comment (TeruyaHaroldo) [inline README.md:59]: Is it not `string`?

File: README.md [hunk 9]
```diff
@@ -29,11 +29,36 @@ Describe any usage specifics for your plugin. Give examples for Android, iOS, An
...
+## Events
+
+| Event | Parameters | Description |
+|-|-|-|
+| **`faceImageCreatedEvent`** | `count: number, total: number, imagePath: String` | Emit when the camera save an image face.  
+| **`faceDetectedEvent`** | `faceDetected: Boolean` | Emit when a face is detected or hided.  
+| **`endCaptureEvent`** | - | Emit when the number of images saved is equal of the number of images set.   
+| **`barcodeScannedEvent`** | `content: String` | Emit content when detect a barcode.   
+| **`errorEvent`** |`error: String` | Emit message error.  
+| **`messageEvent`** | `message: String` | Emit message.
```
Comment (TeruyaHaroldo) [inline README.md:60]: Is it not `string`?

**Review by TeruyaHaroldo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Goulartvic) [reply README.md:63]: Events documentation has no need?

**Review by TeruyaHaroldo — APPROVED** [review]

**Pull request merged**